Game/Source/GDKTestGyms/GDKTestGyms.Build.cs

```diff
@@ -19,6 +19,7 @@ public GDKTestGyms(ReadOnlyTargetRules Target) : base(Target)
 				"GameplayAbilities",
 				"GameplayTasks",
 				"InputCore",
+				"Slate",
```

Comment:
Needed for the input component stuff and binding keys directly.
Maybe I could setup action mappings instead....